docs/02_installation.md

```diff
@@ -7,39 +7,46 @@ EOSIO.CDT currently supports Mac OS X brew, Linux x86_64 Debian packages, and Li
 **If you have previously installed EOSIO.CDT, run the `uninstall` script (it is in the directory where you cloned EOSIO.CDT) before downloading and using the binary releases.**
 
 ## Mac OS X Brew Install
+
 ```sh
 $ brew tap eosio/eosio.cdt
```

Comment:
consider removing the shell prompt $ from all these shell commands

Reply:
done